### PR TITLE
Css fw

### PR DESCRIPTION
**Pull request by bjornhaavard**
This is my pull request. In need of some review to make this better, i am not sure how to build these kind of sites.

### CODE REVIEW

File: feed/index.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2"
+      crossorigin="anonymous"
+    ></script>
+    <title>Alien Social | Feed</title>
+    <link rel="stylesheet" href="../dist/css/index.css" />
+  </head>
+  <body class="d-flex flex-column h-100">
+    <header>
+      <nav class="navbar navbar-expand-lg bg-body-tertiary rounded" aria-label="Eleventh navbar example">
```
Comment (Fermain) [inline feed/index.html]: Please update aria values here. Do you want a blind user to hear "Eleventh navbar example" when they are trying to understand your site?

File: feed/index.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        <div class="container-fluid">
+          <a class="navbar-brand mt-3" href="/">👽</a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarsExample09"
```
Comment (Fermain) [inline feed/index.html]: Doesn't hurt to rename these too

File: feed/index.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+              <input class="form-control" type="search" placeholder="Search" aria-label="Search" />
+            </form>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="flex-shrink-0">
+     
+      <section>
+        <div class="container px-4 py-5" id="custom-cards">
```
Comment (Fermain) [inline feed/index.html]: Generally speaking you should avoid changing the px of a container. Container classes have a specific amount of padding by default and it's better to override these in SASS than to modify them with additional classes.

File: feed/index.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+          <h1 class="pb-2 border-bottom">Posts</h1>
+          <button type="button" id="new-post-button" class="btn btn-primary  " data-bs-toggle="modal" data-bs-target="#exampleModal">New Post</button>
```
Comment (Fermain) [inline feed/index.html]:
```suggestion
          <button type="button" id="new-post-button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">New Post</button>
```

Comment (Fermain) [reply feed/index.html]: Consider renaming exampleModal to something more helpful

File: index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+              </ul>
+              <!-- <form role="search">
+                <input class="form-control" type="search" placeholder="Search" aria-label="Search" />
+              </form> -->
+            </div>
+          </div>
+        </nav>
+      </header>
+    <main class="flex-shrink-0">
+      <section class="h-100 gradient-custom-2">
```
Comment (Fermain) [inline index.html:43]: Indentation is weird, please run a formatter over your project. You can install `prettier` and add this to package.json:

```
npm i -D prettier
```

```json
{
  "scripts": {
    "format": "prettier --write --ext '.html' ."
  }
}
```

`npm run format`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    <div class="container col-xl-10 col-xxl-8 px-4 py-5">
+      <div class="row align-items-center g-lg-5 py-5">
+        <div class="col-lg-7 text-center text-lg-start">
+          <h1 class="display-4 fw-bold lh-1 text-body-emphasis mb-3">Alien Social</h1>
+        </div>
+        <div class="col-md-10 mx-auto col-lg-5">
+          <form class="p-4 p-md-5 border rounded-3 bg-body-tertiary" action="./">
+            <div class="form-floating mb-3">
+              <input type="email" class="form-control" id="floatingInput" placeholder="name@example.com" />
```
Comment (Fermain) [inline index.html:52]:
```suggestion
              <input type="email" class="form-control" id="auth-email" placeholder="name@example.com" required />
```

Comment (Fermain) [inline index.html:52]:
```suggestion
              <input type="email" class="form-control" id="auth-email" placeholder="name@example.com" />
```

File: index.html [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+              <label for="floatingInput">Email address</label>
+            </div>
+            <div class="form-floating mb-3">
+              <input type="password" class="form-control" id="floatingPassword" placeholder="Password" minlength="8" required />
+              <label for="floatingPassword">Password</label>
+            </div>
+            <div class="checkbox mb-3">
+              <label> <input type="checkbox" value="remember-me" /> Remember me </label>
+            </div>
+            <button class="w-100 btn btn-lg btn-primary"" type="submit">Sign up</button>
```
Comment (Fermain) [inline index.html:62]:
```suggestion
            <button class="w-100 btn btn-lg btn-primary" type="submit">Sign up</button>
```

File: profile/index.html [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+            <form role="search">
+              <input class="form-control" type="search" placeholder="Search" aria-label="Search" />
+            </form>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main class="flex-shrink-0">
+      <!-- <section class="container px-4 py-5 h-100 gradient-custom-2">
```
Comment (Fermain) [inline profile/index.html]: If you don't need it, delete it

File: profile/index.html [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+                  <li class="list-group-item">CSS</li>
+                  <li class="list-group-item">JavaScript</li>
+                  <li class="list-group-item">Bootstrap</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="container mt-4">
```
Comment (Fermain) [inline profile/index.html]: I don't think you need two containers here

File: profile/index.html [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+        <div class="row">
+          <div class="col-md-0 d-flex justify-content-end">
+            <div class="card">
+              <div class="card-body">
+                <button type="button" class="btn btn-primary mx-2" onclick="window.location.href=`/index.html`">Log out</button>
```
Comment (Fermain) [inline profile/index.html]:
```suggestion
                <a href="/" class="btn btn-primary mx-2">Log out</a>
```

File: profile/index.html [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+                <button class="btn btn-primary btn-block">Follow</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- Recent posts section -->
+      <div class="container mt-4">
```
Comment (Fermain) [inline profile/index.html]: No need for a second container, move the margin class to the row instead.

File: src/scss/_posts.scss [hunk 12]
```diff
@@ -0,0 +1,13 @@
+.post {
+  border: solid 2px rgb(13, 143, 13);
```
Comment (Fermain) [inline src/scss/_posts.scss:2]: Color values belong in variables and should use the theme system

File: src/scss/_posts.scss [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+
+  &:hover {
+    border-color: white;
+    transition: all 0.5s ease-out;
+  }
+}
+
+.post-images-small {
+  max-width: 300px;
+  height: auto;
```
Comment (Fermain) [inline src/scss/_posts.scss]:
```suggestion
  @extend .h-auto;
```

This is more SASSY

File: src/scss/_posts.scss [hunk 14]
```diff
@@ -0,0 +1,13 @@
+.post {
+  border: solid 2px rgb(13, 143, 13);
+
+  &:hover {
+    border-color: white;
```
Comment (Fermain) [inline src/scss/_posts.scss:5]:
```suggestion
    border-color: $white;
```

File: src/scss/_posts.scss (hunk 12 above)
Comment (Fermain) [reply src/scss/_posts.scss:2]:
```suggestion
  border: solid 2px $primary;
```

File: src/scss/_variables.scss [hunk 15]
```diff
@@ -0,0 +1 @@
+$primary: rgb(13, 143, 13);
```
Comment (Fermain) [inline src/scss/_variables.scss]: This is the absolute bare minimum in terms of Bootstrap customisation. I think you can do more!

File: src/scss/index.scss [hunk 16]
```diff
@@ -1 +1,4 @@
-
+@import "./variables";
```
Comment (Fermain) [inline src/scss/index.scss]:
```suggestion
@import "variables";
```

Same thing, but looks nicer

File: src/scss/index.scss [hunk 17]
```diff
@@ -1 +1,4 @@
...
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
```
Comment (Fermain) [inline src/scss/index.scss]:
```suggestion
@import "../../node_modules/bootstrap/scss/bootstrap";
```

**Review by Fermain — CHANGES_REQUESTED** [review]
There are no show-stoppers here, I haven't viewed the site itself so this is a pure code review. No comments about layout design, just looking for best practice issues and requirements.

File: feed/index.html (hunk 1 above)
Comment (bjornhaavard) [reply feed/index.html]: done!

File: index.html (hunk 5 above)
Comment (bjornhaavard) [reply index.html:43]: I have used prettier all the time, also installed the script.

File: feed/index.html (hunk 2 above)
Comment (bjornhaavard) [reply feed/index.html]: done

File: feed/index.html (hunk 3 above)
Comment (bjornhaavard) [reply feed/index.html]: fixed

File: src/scss/_posts.scss (hunk 13 above)
Comment (bjornhaavard) [reply src/scss/_posts.scss]: changed, and also, new profile page

File: src/scss/_variables.scss (hunk 15 above)
Comment (bjornhaavard) [reply src/scss/_variables.scss]: more added

File: src/scss/index.scss (hunk 16 above)
Comment (bjornhaavard) [reply src/scss/index.scss]: changed

File: src/scss/index.scss (hunk 17 above)
Comment (bjornhaavard) [reply src/scss/index.scss]: also changed

File: src/scss/_posts.scss (hunk 14 above)
Comment (bjornhaavard) [reply src/scss/_posts.scss:5]: done

**Pull request merged**